packages/utilities/psammead-assets/README.md

```diff
@@ -12,13 +12,13 @@ This package provides a collection of common assets that are likely to be requir
 ```jsx
 npm install @bbc/psammead-assets --save
 ```
-
 ## Usage
```

Comment:
Can you please add some details about what `news` is for example, that its a group with a ratio, and add a description of how you would use it, how you could calculate the width & height for viewport etc